.github/workflows/main.yml

```diff
@@ -2,7 +2,6 @@ name: 🚀
 
 on:
   push: { branches: [main] }
-  pull_request: {}
```

Comment:
We should probably keep this, since this is still giving half of our current contributors security to merge. Imo it might be worth merging with failed workflow with non-core-contributors open PRs.

Reply:
I'll dig in, maybe there's a way to disable it only for "external" PRs

Reply:
There weren't any simple settings to disable it for external PRs.

I achieved the desired result by adding an `if` to the test job that checks if one of the secrets are present, skipping otherwise.

Note that unfortunately job-level `if`s can't access secrets directly, so I had to add another preliminary job (`checkEnv`) that does the actually check that can be used in the `if`.

You can see the check was skipped for this PR, but run in the source repo (for the same commit) where the secrets were available: https://github.com/bilalakil/steam-deploy/actions/runs/1626283679